### PR TITLE
Conditional HTTPS hint on login page

### PR DESCRIPTION
**Pull request by DL6ER**
# What does this implement/fix?

Show HTTPS hint on login page only when HTTPS is actually served. If it is served on a port != 443, add this port as well to the link

Depends on FTL in https://github.com/pi-hole/FTL/pull/1653

**Related issue or feature (if applicable):** N/A

**Pull request in [docs](https://github.com/pi-hole/docs) with documentation (if applicable):** N/A

---
**By submitting this pull request, I confirm the following:** 

1. I have read and understood the [contributors guide](https://docs.pi-hole.net/guides/github/contributing/), as well as this entire template. I understand which branch to base my commits and Pull Requests against. 
2. I have commented my proposed changes within the code.
3. I am willing to help maintain this change if there are issues with it later.
4. It is compatible with the [EUPL 1.2 license](https://opensource.org/licenses/EUPL-1.1)
5. I have squashed any insignificant commits. ([`git rebase`](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html))

## Checklist:

- [x] The code change is tested and works locally.
- [x] I based my code and PRs against the repositories `developmental` branch.
- [x] I [signed off](https://docs.pi-hole.net/guides/github/how-to-signoff/) all commits. Pi-hole enforces the [DCO](https://docs.pi-hole.net/guides/github/dco/) for all contributions
- [x] I [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) all my commits. Pi-hole requires signatures to verify authorship
- [x] I have read the above and my PR is ready for review.

### CODE REVIEW

**Review by yubiuser — DISMISSED** [review]
Approving, but don't merge before the relevant FTL is merged. (Maybe changes here are required.)

I tried with ssl port 443, 8443 and non and it works as expected.

File: scripts/pi-hole/js/login.js
Comment (rdwebdesign) [inline scripts/pi-hole/js/login.js]: Before Spellcheck complains...

```suggestion
  // Get information about HTTPS port and DNS status
```

**Review by rdwebdesign — DISMISSED** [review]

**Review by yubiuser — APPROVED** [review]

**Pull request merged**
